tests/TPCTests/test.sh

```diff
@@ -64,7 +64,7 @@ declare -A hosts_http=(
 
 setup_scitokens() {
 	if ! ${XRDSCITOKENS_CREATE_TOKEN} "${XRDSCITOKENS_ISSUER_DIR}"/issuer_pub_1.pem "${XRDSCITOKENS_ISSUER_DIR}"/issuer_key_1.pem test_1 \
-		"https://localhost:7095/issuer/one" "storage.modify:/ storage.create:/ storage.read:/"> "${PWD}/generated_tokens/token"; then
+		"https://localhost:7095/issuer/one" "storage.modify:/ storage.create:/ storage.read:/" 120 > "${PWD}/generated_tokens/token"; then
```

Comment:
Can we make the default 5min? I suspect on slow systems 2min might still not be enough, especially if other tests run between the token being created and the actual TPC tests. If the default is 5min, then you also don't need to pass it in here. Later you could add proper command line argument parsing with [`getopt(3)`](https://www.man7.org/linux/man-pages/man3/getopt.3.html) to this command (i.e. parse `--issuer`, `--validity`, etc).

Reply:
I did not want to change the default so as not to disturb the existing tests that might depend on it being a 1 minute.
However, I can change the token lifetime here to 5 mins if you prefer?

Reply:
Ok, sounds reasonable. I think it looks ready to merge now.